packages/ssx-authjs/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "ssx-authjs",
+  "version": "1.0.0",
```

Use `0.x.0` for a new package and add a changeset with a major version bump. This will ensure this package is bumped to `1.0.0` upon changeset release
```suggestion
  "version": "0.1.0",
```